app/src/main/java/net/vonforst/evmap/api/nobil/NobilModel.kt

```diff
@@ -111,7 +112,10 @@ data class NobilChargerStation(
             chargepoints,
             null,
             "",
-            null,
+            when (chargerStationData.landCode) {
+                "SWE" -> "https://www.energimyndigheten.se/klimat/transporter/laddinfrastruktur/registrera-din-laddstation/elbilsagare/"
+                else -> "mailto:post@nobil.no?subject=" + URLEncoder.encode("Regarding charging station " + chargerStationData.internationalId, "UTF-8").replace("+", "%20")
```

Comment:
Is the `.replace("+", "%20")` necessary? `Uri.encode` might also work as an alternative

Reply:
Uri.encode works better. I switched.